.ci/update_formula_version.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail
```

Comment:
minor: would suggest running this through shellcheck to catch any issues.

Reply:
Found two issues, thanks. I have fixed those. There is other warning which can be ignored in our case as we are using it in sed. https://github.com/koalaman/shellcheck/wiki/SC1004